apps/builder/app/routes/asset.image.$name.ts

```diff
@@ -0,0 +1,35 @@
+import { createReadStream } from "node:fs";
+import { join } from "node:path";
+import { type LoaderArgs, Response } from "@remix-run/node";
+import env from "~/env/env.server";
+
+// this route used as proxy for images to cloudflare endpoint
+// https://developers.cloudflare.com/fundamentals/get-started/reference/cdn-cgi-endpoint/
+
+export const loader = async ({ params, request }: LoaderArgs) => {
+  const { name } = params;
+  if (name === undefined) {
+    throw Error("Name is undefined");
+  }
+
+  const url = new URL(request.url);
+  const width = url.searchParams.get("width");
+  const quality = url.searchParams.get("quality");
+  const format = url.searchParams.get("format");
+  if (width === null || quality === null || format === null) {
+    throw Error("Options are invalid");
+  }
+  const options = `width=${width},quality=${quality},format=${format}`;
+
+  if (env.RESIZE_ORIGIN !== undefined) {
+    const assetUrl = `${env.ASSET_BASE_URL}${name}`;
```

Comment:
`ASSET_BASE_URL` plz add comment with possible defaults in env ts file 
/**
* Possible values are `https://assets-dev....`
*//